target/scripts/startup/setup-stack.sh

```diff
@@ -1144,7 +1144,10 @@ function _setup_fail2ban
   _log 'debug' 'Setting up Fail2Ban'
   if [[ ${FAIL2BAN_BLOCKTYPE} != 'reject' ]]
   then
-    echo -e '[Init]\nblocktype = DROP' >/etc/fail2ban/action.d/iptables-common.local
+    cat >/etc/fail2ban/action.d/nftables-common.local << "EOM"
+[Init]
+blocktype = drop
+EOM
```

Comment:
I did not change it to a HERDOC because of stylistic reasons, but because I figured we may want to add more options here in the future, which would make the line longer and therefore justify a HEREDOC.

I will change it back to `echo ...`.

Maybe giving clear styling rules in such cases is a good idea. If would not like to do it myself as I have already imposed much of my style upon this project.

Reply:
> , but because I figured we may want to add more options 

Ack, that's a valid reason 👍 But then I still prefer to keep it, until we really add more options.

BTW/Did you know, you can also have nice formatted/indented heredocs by adding a `-` --> `>>>-"EOM"`

```bash
#!/bin/bash

# useless if, just to have indentation
if [ foo = foo ]; then
	# just to have s
	cat <<-"EOF"
		#!/bin/bash
		
		echo "hello world"
	EOF
fi
```

However, this is only possible when using tabs for indentation and not spaces 😞

Reply:
> Ack, that's a valid reason 👍🏼 But then I still prefer to keep it, until we really add more options.

Yep, I thought so too :D

---

> However, this is only possible when using tabs for indentation and not spaces disappointed

I knew about this option, but you're right, this only works with tabs...

---

I will resolve this conversation - I think this matter is solved :)